### PR TITLE
DDF-3033 Ensure SecurityManager service is available before trying to use it in ServiceManagerProxy

### PR DESCRIPTION
**Pull request by clockard**
#### What does this PR do?
The issue of the ServiceManagerProxy getting a NPE when trying to get the system subject became very apparent when working on the karaf upgrade ticket. We did find that it is also occasionally happening right now on master as well though. This pr should protect against this.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@mcalcote @emmberk 
#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).

[Test](https://github.com/orgs/codice/teams/test)
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@lessarderic
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)
Verify build passes
#### What are the relevant tickets?
[DDF-3033](https://codice.atlassian.net/browse/DDF-3033)
#### Checklist:
- [ ] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [X] Update / Add Integration Tests

### CODE REVIEW

**Comment by clockard** [comment]
build now

File: distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/ServiceManagerProxy.java [hunk 1]
```diff
@@ -38,7 +43,11 @@ public ServiceManagerProxy(ServiceManager serviceManager) {
 
     @Override
     public Object invoke(Object proxy, Method method, Object[] args) throws Throwable {
-
+        //wait until the security manager is available otherwise the getSystemSubject command will fail
```
Comment (shaundmorris) [inline distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/ServiceManagerProxy.java]: nice comment

File: pom.xml [hunk 2]
```diff
@@ -86,6 +86,7 @@
         <mockito.version>1.10.19</mockito.version>
         <hamcrest-all.version>1.3</hamcrest-all.version>
         <junit.version>4.12</junit.version>
+        <awaitility.version>3.0.0</awaitility.version>
```
Comment (shaundmorris) [inline pom.xml]: Do we expect this to only be an itests? There may have been a decision to keep these properties up in the root pom.

Comment (clockard) [inline pom.xml]: I added a version number in the root pom and replaced the versions with it

**Pull request merged**